src/services/listCourtCases.ts

```diff
@@ -104,6 +105,18 @@ const listCourtCases = async (
     })
   }
 
+  if (allocatedToUserName) {
```

Comment:
It would be better to use this syntax, so we use typeorm function rather than passing SQL. More importantly, this is sanitised.
```
  if (allocatedToUserName) {
    query.andWhere(
      new Brackets((qb) => {
        qb.where({ errorLockedByUsername: allocatedToUserName }).orWhere({
          triggerLockedByUsername: allocatedToUserName
        })
      })
    )
  }
```

Reply:
done